Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -555,34 +555,13 @@ public void Init(IEOSCoroutineOwner coroutineOwner)
                 Init(coroutineOwner, EOSPackageInfo.ConfigFileName);
             }
 
-            //-------------------------------------------------------------------------
-            private EOSConfig LoadEOSConfigFileFromPath(string eosFinalConfigPath)
-            {
-                string configDataAsString = "";
-#if UNITY_ANDROID && !UNITY_EDITOR
-                configDataAsString = AndroidFileIOHelper.ReadAllText(eosFinalConfigPath);
-#else
-                if (!File.Exists(eosFinalConfigPath))
-                {
-                    throw new Exception("Couldn't find EOS Config file: Please ensure " + eosFinalConfigPath +
-                                        " exists and is a valid config");
-                }
-
-                configDataAsString = File.ReadAllText(eosFinalConfigPath);
-#endif
-                var configData = JsonUtility.FromJson<EOSConfig>(configDataAsString);
-
-                print("Loaded config file: " + configDataAsString);
-                return configData;
```

Comment:
This logic is moved to the `FileUtility` class's `ReadAllText` method, which deals with the change in implementation that is unique to Android.